test-case/check-fw-echo-reference.sh

```diff
@@ -84,17 +75,20 @@ do
         fi
         # run echo reference test
         dlogc "alsabat -P $pb_dev --standalone -c $channel -f $fmt -r $rate -n $frames -F $frequency"
-        alsabat -P "$pb_dev" --standalone -c "$channel" -f "$fmt" -r "$rate" -n "$frames" -F "$frequency" &
+        alsabat -P "$pb_dev" --standalone -c "$channel" -f "$fmt" -r "$rate" -n "$frames" -F "$frequency" & alsabatPID=$!
```

Comment:
I'd rather have the alsabatPID assignment on a separate line

Reply:
@marc-hb no, just because it's a new command, don't remember ever seeing `cmd1 & cmd2` in one line in shell. I understand trying to keep `$!` directly attached to the previous command to avoid them being ever separated by accident, so, I'm not strongly against this, just looks weird to me.

Reply:
BTW I just checked the documentation about this (because of some unrelated review): `&` and `;` have the same precedence.